### PR TITLE
exec: honor --privileged

### PR DESCRIPTION
**Pull request by giuseppe**
write the capabilities to the configuration passed to the OCI
runtime.

Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>

### CODE REVIEW

File: libpod/oci_conmon_exec_linux.go [hunk 1]
```diff
@@ -387,7 +387,7 @@ func (r *ConmonOCIRuntime) startExec(c *Container, sessionID string, options *Ex
 		finalEnv = append(finalEnv, fmt.Sprintf("%s=%s", k, v))
 	}
 
-	processFile, err := prepareProcessExec(c, options.Cmd, finalEnv, options.Terminal, options.Cwd, options.User, sessionID)
+	processFile, err := prepareProcessExec(c, options.Cmd, options.CapAdd, finalEnv, options.Terminal, options.Cwd, options.User, sessionID)
```
Comment (rhatdan) [inline libpod/oci_conmon_exec_linux.go:390]: Can you change this interface to just pass in the options?

**Review by rhatdan — APPROVED** [review]
This also needs tests added.

File: libpod/oci_conmon_exec_linux.go (hunk 1 above)
Comment (giuseppe) [reply libpod/oci_conmon_exec_linux.go]: done

**Comment by rhatdan** [comment]
LGTM

**Comment by rhatdan** [comment]
@containers/podman-maintainers PTAL

**Review by vrothberg — COMMENTED** [review]
LGTM

**Pull request merged**